### ISSUE TITLE
Crash when resolution is 1440p

### ISSUE THREAD

**Issue by tjolle** [veldenb/plugin.program.moonlight-qt#33]
I’m running LibreELEC 10.0.4 on a Raspberry Pi 4.

When I set the EGL-resolution to 1440p, moonlight-qt seems to just crash as soon as the client starts. The issue doesn’t appear when I force 1080p, or when I set it to Default and let it run in 4k. The log doesn’t produce anything useful but I’ll add it to the issue if you want.

**Comment by veldenb** [comment]
Hi,

Does setting 1440p in Kodi itself work?

**Comment by tjolle** [comment]
Yes - I also forced a 1440p output through cmdline.txt, and that worked fine in Kodi but same result with moonlight.Curiously I noticed that setting the EGL-resolution also put it in 120Hz, and I was thinking maybe that had something to do with it, but when I used cmdline.txt I also managed to put it at 60Hz and it made no difference.I’m wondering if this is an issue upstream?10 mars 2023 kl. 11:50 skrev veldenb ***@***.***>:﻿
Hi,
Does setting 1440p in Kodi itself work?

**Comment by veldenb** [comment]
Not that I'm aware of, on my own TV switching modes works fine. Have you tried a different TV or display?

**Comment by LogicDiscovery** [comment]
I was about to test on a different display when I tried a last ditch effort and it looks like the culprit was me having `hdmi_enable_4kp60=1 ` in my config.txt. Once I removed it and forced 2560x1440 through cmdline.txt (otherwise LibreELEC defaulted to 4096x2160) everything has worked as expected so far.

You can close this issue if you don’t have anything else to add before then. Thanks!
